src/hid/hid_device.cc

```diff
@@ -212,14 +212,9 @@ Status HidDevice::Init() {
 }
 
 Status HidDevice::Wink() {
-  uint8_t cmd = kCtapHidWink;
-  OK_OR_RETURN(SendCommand(cmd, std::vector<uint8_t>()));
-
   std::vector<uint8_t> recv_data;
-  Status status = ReceiveCommand(kReceiveTimeout, &cmd, &recv_data);
-  if (cmd != kCtapHidWink) return Status::kErrInvalidCommand;
-  if (!recv_data.empty()) return Status::kErrInvalidLength;
-  return status;
+  return SendRecvCommand(kCtapHidWink, std::vector<uint8_t>(), &recv_data, true,
```

Comment:
Does that mean `Wink` can now process Keepalive packets? Looks like a change in existing behaviour, and not just an extension to allow to send raw data.